tools/dashboard-plugin-template/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+    "_note": "This file is only for IDE. For build use tsconfig.[dev/build].json (dev mode is less strict)",
+    "compilerOptions": {
+        "declaration": true,
```

Comment:
missing `strict`. methinks the skels don't have it and probably should..

Reply:
Good catch, updated in both the pacakages and skeletons